shell/platform/linux/fl_view_accessible.cc

```diff
@@ -36,40 +41,93 @@ static FlEngine* get_engine(FlViewAccessible* self) {
   return self->engine;
 }
 
+static FlAccessibleNode* create_node(FlViewAccessible* self,
+                                     const FlutterSemanticsNode* semantics) {
+  FlAccessibleNode* node = nullptr;
+  FlEngine* engine = get_engine(self);
+  if (semantics->flags & kFlutterSemanticsFlagIsTextField) {
+    node = fl_accessible_text_field_new(engine, semantics->id);
```

Comment:
Might be simpler/safer to just return the value here rather than potentially leak it later if this function was changed in the future to return after the if branch.